README.md

```diff
@@ -219,80 +219,58 @@ The corresponding function in on Tarantool Cartridge router may look like (uses
 
 See more examples in the module tests.
 
-#### Call stored functions in Tarantool instance
-
-You can bind repository methods to calls of the stored functions in the Tarantool instance using the `@Query` annotation
-with the stored function name specified in the `functionName` parameter.
-For such methods, you can specify the stored function response format so that it will be parsed correctly. The response
-format may be either an object (and a list of objects) or a tuple (and a list of tuples).
-
-```java
-@Data
-@Tuple
-public class SampleUser {
-    private String name;
-    private String lastName;
-}
-```
+##### Specify output
```

Comment:
The previous header and paragraph here was more clear to me. It tells about a use-case.

Reply:
If you mean the previous one - the deleted paragraph, it was talking about the same thing as above. Those we have 2 paragraphs in the master talking about the same thing, only with different examples and this is more confusing, even take the same headings. `Proxy methods` and `Call stored functions`

![image](https://user-images.githubusercontent.com/22258443/152529986-db2fa031-95f3-489d-b033-359386266e2f.png)

Reply:
Ah OK, I see. I wonder how it got duplicated +_+
BTW, here was a description of `functionName` parameter and it seems to be missing now.

Reply:
Added a few words about `functionName` as it was in master